package.json

```diff
@@ -72,26 +73,28 @@
     "test-client-wp": "karma start test/client/webpack/karma.conf.js",
     "test-client-wp-ci": "karma start --browsers PhantomJS,Firefox test/client/webpack/karma.conf.coverage.js",
     "test-client-wp-cov": "karma start test/client/webpack/karma.conf.coverage.js",
-    "test-client": "npm run test-client-rjs && npm run test-client-wp",
-    "test-client-ci": "npm run test-client-rjs-ci && npm run test-client-wp-ci",
-    "test-client-cov": "npm run test-client-rjs-cov && npm run test-client-wp-cov",
-    "test-func": "magellan",
-    "test-func-cov": "magellan --config=magellan-cov.json",
-    "test-func-raw": "mocha --opts test/func/mocha.opts test/func/spec/",
-    "test-func-raw-cov": "mocha --opts test/func/mocha-cov.opts test/func/spec/",
-    "test-func-sauce": "magellan --sauce --create_tunnels --config=magellan-cov.json --browser=chrome_latest_Windows_2012_R2_Desktop,firefox_latest_Windows_2012_R2_Desktop,IE_11_Windows_2012_R2_Desktop,IE_8_Windows_2008_Desktop,IE_9_Windows_2008_Desktop,IE_10_Windows_2012_Desktop",
-    "test-func-sauce-safari": "magellan --serial --sauce --create_tunnels --config=magellan-sauce-safari.json --browser=safari_7_OS_X_10_9_Desktop",
-    "test-func-ci": "magellan --config=magellan-cov.json --browsers=phantomjs,firefox",
-    "test": "npm run test-client && npm run test-func",
-    "test-ci": "npm run test-client-ci && npm run test-func-ci",
+    "test-client": "builder concurrent --buffer test-client-rjs test-client-wp",
+    "test-client-ci": "builder concurrent --buffer test-client-rjs-ci test-client-wp-ci",
+    "test-client-cov": "builder concurrent --buffer test-client-rjs-cov test-client-wp-cov",
+    "test-func": "mocha --color --opts test/func/mocha.opts test/func/spec/",
+    "test-func-cov": "mocha --color --opts test/func/mocha-cov.opts test/func/spec/",
+    "test-func-ci": "builder run test-func-cov",
+    "test": "builder run test-client && builder run test-func",
+    "test-ci": "builder concurrent --buffer test-client-ci test-func-ci",
+    "setup-local-selenium": "selenium-standalone start >/dev/null 2>&1",
+    "setup-local": "builder run setup-local-selenium",
+    "test-func-local": "sleep 5 && echo \"Starting ${ROWDY_SETTINGS}\" && builder run test-func-cov",
+    "setup-sauce-connect": "node_modules/sauce-connect-launcher/sc/*/bin/sc",
+    "setup-sauce": "builder run setup-sauce-connect",
+    "test-func-sauce": "sleep 20 && echo \"Starting ${ROWDY_SETTINGS}\" && builder run test-func-cov",
```

Comment:
And separately, (documented in `requirepack`, but not here), if you wanted to you could:

In one terminal:

```
$ builder run setup-sauce
```

leave it going. In another terminal:

```
$ builder run test-func
```

without waiting.

Ultimately, I'm not that concerned with dev's doing a ton of local sauce connect from their dev laptops, since we're unlikely to change the suite / code that significantly at this point. my main goals here are: (1) speed up CI, and (2) actually get _all_ the Safari Sauce labs tests running (we only previously ran one due to magellan + sauce connect limitations with repeated setup calls)